<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tool_BinMergeClass</class>
 <widget class="QMainWindow" name="Tool_BinMergeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool_BinMerge</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnSelectFiles">
          <property name="text">
           <string>选择文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectDir">
          <property name="text">
           <string>选择目录</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="btnTrans">
          <property name="text">
           <string>转换</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbList">
          <item>
           <property name="text">
            <string>Pcap</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Binary</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnLvxTrans">
        <property name="text">
         <string>提取数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListView" name="lstDisplay">
        <property name="movement">
         <enum>QListView::Static</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pgbStatus">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QOpenGLWidget" name="ogwDisplay">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Tool_BinMerge.qrc"/>
 </resources>
 <connections/>
</ui>
